app/validators/routes_and_redirects_validator.rb

```diff
@@ -166,7 +166,7 @@ def internal?(destination)
     end
 
     def government_domain?(host)
-      host.end_with?(".gov.uk") || host.end_with?(".judiciary.uk")
+      host.end_with?(".gov.uk") || host.end_with?(".judiciary.uk") || host.end_with?("etl.beis.gov.uk")
```

Comment:
Minor: looks like you can use `end_with?` with multiple arguments which would make this statement a bit more readable -> https://apidock.com/ruby/String/end_with%3F

Reply:
TIL!

Reply:
Ooh, thanks, for the pointer.  I'll update to use this.